Routine/Service/ServiceHttpHandler.cs

```diff
@@ -393,18 +403,45 @@ private void IndexApplicationModelIfNecessary()
 			}
 		}
 
-		private void AddResponseHeaders(HttpContext context)
+		private void AddResponseHeaders()
 		{
 			foreach (var responseHeader in serviceContext.ServiceConfiguration.GetResponseHeaders())
 			{
 				var responseHeaderValue = serviceContext.ServiceConfiguration.GetResponseHeaderValue(responseHeader);
 				if (!string.IsNullOrEmpty(responseHeaderValue))
 				{
-					context.Response.Headers.Add(responseHeader, HttpUtility.UrlEncode(responseHeaderValue));
+					httpContextBase.Response.Headers.Add(responseHeader, HttpUtility.UrlEncode(responseHeaderValue));
 				}
 			}
 		}
 
+		internal void SetHttpContextBase(HttpContextBase httpContextBase)
```

Comment:
Bunu test için yazdın muhtemelen. Kurtulmak için HttpContext'li interface method'unu explicit implement et. public ProcessRequest'i böylece HttpContextBase alacak hale getirebilirsin. Öyle olunca da test'lerde direkt HttpContextBase alan ProcessRequest'i çağırabilir hale gelirsin ve bu method'a gerek kalmaz

Reply:
Evet test için yazmıştım.🙉 ilgili düzeltmeyi sağlarım.